larray/core/axis.py

```diff
@@ -274,6 +274,64 @@ def extend(self, labels):
         labels = self._length + other._length if self.iswildcard else np.append(self.labels, other.labels)
         return Axis(labels, self.name)
 
+    def split(self, sep='_', names=None, regex=None, return_labels=False):
+        """Split axis and returns a list of Axis.
+
+        Parameters
+        ----------
+        sep : str, optional
+            Delimiter to use for splitting. Defaults to '_'.
+
+            When `regex` is provided, the delimiter is only used on `names` if given as one string or on axis name if
```

Comment:
why a blank line? To force line return?

Reply:
no idea. I think it was that way where I copy-paste that block from :) (or I introduced it by mistake)